stable/redmine/templates/deployment.yaml

```diff
@@ -37,7 +37,7 @@ spec:
           {{- end }}
           env:
             {{- if .Values.extraVars }}
-            {{- toYaml .Values.extraVars | indent 12 }}
+            {{- toYaml .Values.extraVars | nindent 12 }}
```

Comment:
`nindent` will add a newline in the yaml,  I would suggest the following instead :

```suggestion
{{- toYaml .Values.extraVars | indent 12 }}
```

Reply:
Are you referring to the blank line added before REDMINE_DB_MYSQL when extraVars is used?

```
{{- toYaml .Values.extraVars | indent 12 }}
```
This still causes the rending error, the  ```{{-``` is causing the template to render on the same line as ```env:```

```
{{ toYaml .Values.extraVars | indent 12 }}
```
Will produce a working render, however doesn't follow the same indentation as the rest of the file.